libs/service/api_service.cpp

```diff
@@ -64,3 +237,162 @@ ebpf_verify_program(
 
     return result;
 }
+
+ebpf_result_t
+ebpf_verify_and_load_program(
+    const GUID* program_type,
+    ebpf_handle_t program_handle,
+    ebpf_execution_context_t execution_context,
+    ebpf_execution_type_t execution_type,
+    uint32_t handle_map_count,
+    fd_handle_map* handle_map,
+    uint32_t byte_code_size,
+    uint8_t* byte_code,
+    const char** error_message,
+    uint32_t* error_message_size)
+{
+    ebpf_result_t result = EBPF_SUCCESS;
+    int error = 0;
```

Comment:
make this use ebpf_result_t instead of int

Reply:
This function returns ebpf_result_t. "int error" is used only to get return code from verifier functions which return 0 or 1.

Reply:
I'm saying returning 0 or 1 from analyze() in verifier_service.cpp is undesirable, and it should instead return ebpf_result_t from that function.

Reply:
Fixed the functions (_analyze() and few others). "error" in this function is now only used to get the win32 error from invoke_ioctl.